apps/api/resources.dev/view/example1.ttl

```diff
@@ -1,25 +1,44 @@
 PREFIX schema: <http://schema.org/>
 prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
 prefix view: <https://cube.link/view/>
+prefix view-builder: <https://view-builder.described.at/>
 
 <> a </api/View>, view:View ;
   schema:identifier "example1" ;
   schema:alternateName "TEST1" ;
-  schema:name "Second view" ;
-  <urn:ssz:source> <#source-bew>, <#source-stf> ;
+  schema:name "Test filtering by term set" ;
+  view-builder:source <#source-bew> ;
   schema:author </user/tpluscode> ;
+  view:dimension <#dim-raum>, <#measure> ;
+  view:filter
+    [
+      a view:Filter ;
```

Comment:
Why not the usual?
	
	<https://cube.link/view/filter> [
		<http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://cube.link/view/Filter> ;
		<https://cube.link/view/dimension> _:b9 ;
		<https://cube.link/view/operation> <https://cube.link/view/Eq> ;
		<https://cube.link/view/argument> <https://ld.stadt-zuerich.ch/statistics/code/R00200> ;
	] ;

Reply:
I don't understand. Which is not the usual? :)